internal/cli/atlas/integrations/integrations_test.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package integrations
+
+import (
+	"testing"
+
+	"github.com/mongodb/mongocli/internal/cli"
+)
+
+func TestBuilder(t *testing.T) {
```

Comment:
this is not an integration test, is still a unit test, also watch out for missing build tags or these tests won't run